config/config.go

```diff
@@ -98,9 +107,38 @@ func (rc *RedisConf) validate() error {
 	if rc.IsSentinel() && rc.MasterName == "" {
 		return errors.New("the master name must not be empty in sentinel mode")
 	}
+	if rc.MaxMemPolicy != ValidRedisMaxMemPolicy {
+		return errors.New("valid maxmempolicy config value must be noeviction, but get " + rc.MaxMemPolicy)
+	}
+	if rc.AppendOnly != ValidRedisAppendOnly {
+		return errors.New("valid appendonly value must be yes, but get " + rc.AppendOnly)
+	}
 	return nil
 }
 
+func processRedisDataPersistConf(rc *RedisConf) {
+	// the sentinel addrs would be split with comma
+	addrs := strings.Split(rc.Addr, ",")
+	cli := redis.NewClient(&redis.Options{
+		Addr:     addrs[0],
+		Password: rc.Password,
+		PoolSize: 1,
+	})
+	defer cli.Close()
+	memPolVal, err := cli.ConfigGet(context.Background(), "maxmemory-policy").Result()
```

Comment:
@Napoleon0621 I think we should get them from the `info` command instead of `config` since the `config` is a sensitive command, so most DBAs will rename this command.

Reply:
@git-hulk Agree on that, done.